src/DotNetBoilerplate.Application/Events/Responses/EventsInfoResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿using DotNetBoilerplate.Application.Events.DTOs;
+using DotNetBoilerplate.Core.Events;
+
+namespace DotNetBoilerplate.Application.Events.Responses;
+
+public class EventsInfoResponse
```

Comment:
możemy zrobić to recordem zamiast class -recordy są idealne do dto i generalnie struktur nie mających zachowań, tylko opierających się na samych wartościach